src/api.v2/routes/qc.js

```diff
@@ -1,4 +1,17 @@
+const { getQCState, runQC, handleResponse } = require('../controllers/qcController');
+
+const { expressAuthorizationMiddleware } = require('../middlewares/authMiddlewares');
+
 module.exports = {
-  'qc#response': () => {
+  'qc#get': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => getQCState(req, res).catch(next),
+  ],
+  'qc#run': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => runQC(req, res).catch(next),
+  ],
+  'qc#response': (req, res, next) => {
+    handleResponse(req, res).catch(next);
```

Comment:
why is it ok to not call the authorization middleware in this case?

Reply:
they are sns messages, we don't create them.

This was already like this in v1, and I think that is because `parseSNSMessage` calls `validator.validate` from this lbirary https://github.com/aws/aws-js-sns-message-validator

Reply:
we could add a ticket to check that this is actually secure to the backlog

Reply:
yes, I think we should evaluate that.